### PR TITLE
Feature: Use `flutter_gen` for assets

### PR DESCRIPTION
**Pull request by yos-syky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,10 +28,12 @@ Icon(
...
+
+Always add line `package: 'aegis'`
+
 ```dart
 Image.asset(
-  AegisAssets.phone,
+  AegisAssets.illustrations_transaction_hourglass.path,
   package: 'aegis',
 ),
 ```
```
Comment (danyrzkwn) [inline README.md:39]: Please add the command to run the runner

File: assets/illustrations/account.png
Comment (danyrzkwn) [inline assets/illustrations/account.png:1]: Please consider to remove all the png files

File: lib/assets.dart [hunk 2]
```diff
@@ -1,3 +1,4 @@
 library assets;
 
-export 'src/assets/assets.dart';
+// export 'src/assets/assets.dart';
+export 'src/generated/assets.gen.dart';
```
Comment (danyrzkwn) [inline lib/assets.dart:4]: Do we still have the usage of previous AegisAssets in the Athena?

Comment (yos-syky) [reply lib/assets.dart:4]: Nope, will be deleted

File: README.md (hunk 1 above)
Comment (yos-syky) [reply README.md]: Done, check new commit

**Review by danyrzkwn — APPROVED** [review]
Great job! Your request was approved. 🥳

🚨 Please follow these instructions before merging this PR.
1. Please make sure **the destination branch is `develop`**
2. Please make sure **the source branch is already rebased** to the last commit of the `develop` branch.
3. Please make sure that all **the changes requested are resolved**.
4. Please make sure **no actions that still running.**
5. Change the merging method to **`Squash and merge`**
6. **Rename the commit message** to the most descriptive message.
7. Confirm the merge and **your task is done**.

Thank you 💰

**Pull request merged**